pandas/core/indexes/base.py

```diff
@@ -2040,18 +2040,47 @@ def isna(self):
 
     def notna(self):
         """
-        Inverse of isna
+        Detect existing (non-missing) values.
+
+        Return a boolean same-sized object indicating if the values are not NA.
+        Non-missing values get mapped to True. Characters such as empty
```

Comment:
``True``

Reply:
Thx for the trick with the double quotes. I'm using it now.